pkgs/by-name/pr/prism-model-checker-unwrapped/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  coreutils,
+  fetchFromGitHub,
+  java,
+  cctools,
+}:
+stdenv.mkDerivation rec {
+  pname = "prism-model-checker-unwrapped";
+  version = "4.8.1";
+
+  src = fetchFromGitHub {
+    owner = "prismmodelchecker";
+    repo = "prism";
+    rev = "v${version}";
+    hash = "sha256-igFRIjPfx0BFpQjaW/vgMEnH2HLC06aL3IMHh+ELB6U=";
+  };
+
+  nativeBuildInputs = [ java ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ cctools ];
+
+  postPatch = ''
+    sed -i "s/\/bin\/mv/mv/" prism/install.sh
```

```suggestion
    substituteInPlace prism/install.sh
      --replace-fail "/bin/mv" "mv"
```

Should be more resilient to failures